tests/test_lexical_parse.py

```diff
@@ -0,0 +1,168 @@
+import y_py as Y
```

Comment:
This is just a better version of the other file. I would just drop the other one and then clean this one up. Can remove the markdown for example.

Could also drop this file completely too. Or use it as the full cycle test like I suggested in the previous comment.

Reply:
Updated it!